domain-management/src/main/java/org/jboss/as/domain/management/security/SecurityRealmService.java

```diff
@@ -117,7 +118,9 @@
  * requiring any of the capabilities provided by the realm.
  *
  * @author <a href="mailto:darran.lofthouse@jboss.com">Darran Lofthouse</a>
+ * @author <a href="mailto:ropalka@redhat.com">Richard Opalka</a>
  */
+// TODO: refactor org.jboss.msc.service.Service to org.jboss.msc.Service when WildFly codebase is fixed
```

Comment:
We can't change anything in this one that would break a subsystem until WF 23. External subsystems can depend on this class and we can't break them.